src/renderer/tvgRender.h

```diff
@@ -160,6 +160,7 @@ struct RenderShape
     RenderColor color{};
     RenderStroke *stroke = nullptr;
     FillRule rule = FillRule::NonZero;
+    bool fillTrim = false;
```

Comment:
Hmm.. I wonder how to figure out the fill for 3 cases: stroke only, fill only, both.
Otherwise, probably, we can put this variable in`TrimPath` like this:
```
enum Target : uint8_t { Stroke, Fill, Both };
```

Reply:
Do we want to support only fill trimming without trimming the stroke at the same time? via the API? The Lottie standard requires trimming both. so what are the use cases?

Please note that, for now, TrimPath is part of RenderStroke. This was motivated by the fact that in lotties fill-only trimming (for shapes without a stroke) doesn’t seem very common, whereas trimming both the stroke and fill-or just the stroke-  is more frequent (I can check numbers tmr). However, if we are considering support for fill-only trimming, I think TrimPath should be moved from RenderStroke to RenderShape. In that case, it should likely be a pointer rather than an object. But then if the stroke is trimmed, two objects will be created-RenderStroke in RenderShape and TrimPath in RenderShape.